modules/core/test/v2/unit/errors.ts

```diff
@@ -4,17 +4,23 @@ import { BitGoJsError } from '../../../src/errors';
 
 describe('Error handling', () => {
 
-  it('should construct custom errors if Error.captureStackTrace is missing', () => {
+  it('should construct custom errors if Error.captureStackTrace is missing', function () {
```

Comment:
Why change these tests to use `function` keyword?

Reply:
arrow functions are generally discouraged in mocha https://mochajs.org/#arrow-functions

Reply:
I am aware of that, but I'm not referring to the test using `this` (or at all), so it shouldn't matter.

Reply:
but you might in the next PR ;)